features/application/Titles.feature

```diff
@@ -38,5 +38,5 @@ Feature: The HTML title is correct on each page
       | /mesh/default/data-planes                | Data Plane Proxies |
       | /mesh/default/data-plane/data-plane-name | Data Plane Proxy   |
 
-      | /mesh/default/policies/circuit-breakers         | CircuitBreaker |
-      | /mesh/default/policy/circuit-breakers/program-0 | CircuitBreaker |
+      | /mesh/default/policies/CircuitBreaker         | CircuitBreaker |
```

Comment:
Has there been any discussion elsewhere on changing these to be camelCase?

Reply:
They’re changing specifically to exact name of policy _types_ rather than using the API path. That makes quite a few things easier across the application and eventually this is necessary to allow us to get rid of the global policy types state (as you would otherwise often need to map from policy type to policy path).

Reply:
@lahabana and I talked about this yesterday. There otherwise isn’t a more formal discussion to refer to, no.

Reply:
Wait... the discussion we had yesterday was around picking the right default in the dropdown. We didn't talk about changing url paths.

Reply:
If we don’t change the URL paths, we cannot avoid the global knowledge of `/policies` because that’s what allows us to map from policy type name to policy type API path.

The whole point of that change was to avoid that dependency.

Reply:
Thats sounds better 👍